pkg/apiserver/auditing/types.go

```diff
@@ -96,6 +96,17 @@ func (a *auditing) K8sAuditingEnabled() bool {
 //
 func (a *auditing) LogRequestObject(req *http.Request, info *request.RequestInfo) *auditv1alpha1.Event {
 
+	// Ignore the dryRun k8s request.
+	if info.IsKubernetesRequest {
+		values := req.URL.Query()
+		if v, ok := values["dryRun"]; ok {
+			if len(v) > 0 && v[0] == v1.DryRunAll {
```

Comment:
I think we simplify this just check the existence of dryRun, same with k8s https://github.com/kubernetes/kubernetes/blob/master/staging/src/k8s.io/apiserver/pkg/endpoints/handlers/rest.go#L421